isic/core/stats.py

```diff
@@ -10,7 +11,7 @@ def get_archive_stats():
         'annotations_count': Annotation.objects.count(),
         'collections_count': Collection.objects.count(),
         'contributors_count': Contributor.objects.count(),
-        'images_count': Image.objects.count(),
+        'images_count': Accession.objects.count(),
```

Comment:
This is not accurate. There could be all sort of garbage files in Accessions which are not really images.